Makefile

```diff
@@ -517,3 +517,31 @@ dev-release:
 validate-release: setup-venv
 	$(Q)$(PYTHON_VENV_DIR)/bin/pip install yq==2.10.0
 	BUNDLE_VERSION=$(BASE_BUNDLE_VERSION) CHANNEL="alpha" ./hack/validate-release.sh
+
+
+## -- Download latest CSV --
+
+SBO_PKG ?= $(OUTPUT_DIR)/service-binding-operator.package.yaml
+latestCSV ?= ""
+latestCSVLoc ?= ""
+latestCSVver ?= ""
+.PHONY: get-latest-csv
+## get-latest-csv
+get-latest-csv: get_sbo_pkg get_csv_version
+	$(eval latestCSVLoc := $(OUTPUT_DIR)/$(latestCSV).clusterserviceversion.yaml)
+	$(Q)wget https://raw.githubusercontent.com/redhat-developer/service-binding-operator-manifests/master/$(latestCSVver)/$(latestCSV).clusterserviceversion.yaml -O $(latestCSVLoc)
+	$(Q)sed -i 's|placeholder|openshift-operators|g' $(latestCSVLoc)
+
+
+.PHONY: get_sbo_pkg
+## get_sbo_pkg
+get_sbo_pkg:
+	$(Q)wget https://raw.githubusercontent.com/redhat-developer/service-binding-operator-manifests/master/service-binding-operator.package.yaml -O $(SBO_PKG)
+
+
+.PHONY: get_csv_version
+## get_csv_version
+get_csv_version:
+	$(eval latestCSV := $(shell grep -wo "service-binding-operator\.v.*" $(SBO_PKG)))
+	$(eval latestCSVver := $(subst service-binding-operator.v, , $(latestCSV)))
```

Comment:
what is the purpose of this? we should not grab "the latest CSV" and deploy it to the cluster, because it might not correspond to the status we have on the branch under test. If we need to deploy SBO CSV (which I highly doubt), we should generate the actual version from the code in the given branch and then deploy it.

Reply:
> what is the purpose of this?

We generate new CSV and push it to service-binding-operator-manifests repo so this target will download the latest CSV pushed. 

> If we need to deploy SBO CSV (which I highly doubt), we should generate the actual version from the code in the given branch and then deploy it.

This test was already there in e2e test suite, so according to ticket says I just converted it to the acceptance test.

Reply:
code removed.